package.json

```diff
@@ -8,45 +8,47 @@
-    "clean": "lerna run clean --parallel",
-    "rebuild": "node ./scripts/invalidate-just-cache.js && npx lerna build --stream",
-    "code-style": "lerna run code-style --stream",
-    "change": "beachball change",
+    "builddemo": "node ./scripts/monorepo/buildTo.js fabric-website-resources",
+    "buildto": "node ./scripts/monorepo/buildTo.js",
+    "bundle": "lerna run bundle --stream",
+    "bundle:prod": "lerna run bundle --stream -- --production",
     "bundlesize": "cd scripts && npm run bundlesize",
     "bundlesizecollect": "cd scripts && just-scripts bundle-size-collect",
+    "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
+    "clean": "lerna run clean --parallel",
+    "codepen": "cd packages/office-ui-fabric-react && node ../../scripts/local-codepen.js",
+    "code-style": "lerna run code-style --stream",
     "create-component": "node scripts/create-component.js",
     "create-package": "node scripts/create-package.js",
     "create-page": "node scripts/create-page.js",
-    "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
+    "change": "beachball change",
+    "dom-test": "cd apps/dom-tests && just-scripts jest-dom-with-webpack",
+    "generate-version-files": "yarn workspace @uifabric/build just generate-version-files",
+    "gulpbuild": "gulp build",
+    "lint": "lerna run lint --parallel",
     "publish:beachball": "beachball publish",
+    "check-for-changed-files": "cd scripts && just-scripts check-for-modified-files",
+    "perf:test": "yarn lerna run --stream --scope @fluentui/perf-test perf:test",
+    "perf:test:base": "yarn lerna run --stream --scope @fluentui/perf-test perf:test:base",
+    "preinstall": "node ./scripts/use-yarn-please.js",
+    "postinstall": "node ./scripts/postinstall.js",
+    "prettier": "node scripts/prettier.js",
+    "rebuild": "node ./scripts/invalidate-just-cache.js && npx lerna build --stream",
+    "scrub": "node ./scripts/scrub.js",
+    "start:legacy": "yarn workspace @uifabric/fabric-website-resources start",
+    "start": "node scripts/start.js",
+    "start-exp": "yarn workspace @uifabric/experiments start",
+    "test": "lerna run test --stream",
     "update-api": "cd packages/office-ui-fabric-react && npm run update-api",
     "update-snapshots": "lerna run update-snapshots --stream",
     "update-a11y": "cd apps/a11y-tests && npm run update-snapshots",
-    "check-for-changed-files": "cd scripts && just-scripts check-for-modified-files",
-    "prettier": "node scripts/prettier.js",
-    "generate-version-files": "yarn workspace @uifabric/build just generate-version-files",
-    "codepen": "cd packages/office-ui-fabric-react && node ../../scripts/local-codepen.js",
-    "dom-test": "cd apps/dom-tests && just-scripts jest-dom-with-webpack",
-    "lint": "lerna run lint --parallel",
-    "test": "lerna run test --stream"
+    "vrtest": "cd apps && cd vr-tests && npm run screener"
```

Comment:
I had to alphabetize this file, it's getting out of control a bit. The meaningful change here is:

```
    "perf:test": "yarn lerna run --stream --scope @fluentui/perf-test perf:test",
    "perf:test:base": "yarn lerna run --stream --scope @fluentui/perf-test perf:test:base",
```

Reply:
shouldn't need to do `yarn lerna run` ... just be `lerna run` is enough

Reply:
Ok, I've removed `perf:test` commands and also `gulp build` and `build:fluent`. Package level perf-test commands will exist for both Fabric and Fluent and won't appear in root `package.json`.